test/correctness/debug_to_file.cpp

```diff
@@ -15,13 +15,13 @@ int main(int argc, char **argv) {
         return 0;
     }
 
-    std::string f_mat = Internal::get_test_tmp_dir() + "f.mat";
```

Comment:
I'm confused - doesn't this remove .mat testing?

Reply:
Yeah, it does so in favor of npy. I could back out this change, or better yet, do both.

Reply:
Yeah I think this should test .mat and .npy